docs/docs/dev_docs/contracts/events.md

```diff
@@ -1,17 +1,104 @@
 ## Events
+Events in Aztec works similarly to Ethereum events in a sense that they are a way for contracts to communicate with the outside world.
+They are emitted by contracts and stored inside each instance ofAztecNode.
+Aztec events are currently represented as raw data and are not ABI encoded.
+ABI encoded events are a feature that will be added in the future.
+
+Unlike on Ethereum, there are 2 types of events supported by Aztec: encrypted and unencrypted.
+
+### Encrypted Events
+Encrypted events can only be emitted by private functions and are encrypted using a public key of a recipient.
+For this reason it is necessary to register a recipient in the AztecRPC server before encrypting the events for them.
```

Comment:
This reminded me that we need to broadcast account data, when an account contract is deployed, so that the public key and partial address can be grabbed automatically by the simulator (from the public node). https://github.com/AztecProtocol/aztec-packages/issues/1778